transport/http/outbound_test.go

```diff
@@ -95,7 +95,7 @@ func TestCreateRequest(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.desc, func(t *testing.T) {
-			o := &Outbound{urlTemplate: defaultURLTemplate}
+			o := &Outbound{urlTemplate: defaultURLTemplate, transport: &Transport{}}
```

Comment:
why this change?

Reply:
I added new field reservedHeaderMetric to transport, so this test fails if transport is nil.